src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int MAX_FIGURE_LENGTH = 7;
+
+    public static void main(String[] args) {
```

Comment:
Don't extend your `Main/Application` class from `FigureSupplier` or `ColorSupplier`. To be able to call the non-static method, we just need to create an instance of the class. Here, it's correctly done by creating an instance of `FigureSupplier`.